src/api/routes/ws/web-socket-transmitter.ts

```diff
@@ -130,6 +137,12 @@ export class WebSocketTransmitter {
     }
   }
 
+  private async bnsImportUpdate(bnsNamesOnchainImported: boolean, bnsSubdomainsImported: boolean) {
+    if (this.channels.find(c => c.hasListeners('bnsImport'))) {
+      await this.send('bnsImport', bnsNamesOnchainImported, bnsSubdomainsImported);
+    }
+  }
+
```

Comment:
`bnsImport` is an event that is only relevant internally to the API, so there's no need to add it as a WebSocket event for external users to consume. You should revert this file (and web-socket-channel.ts)